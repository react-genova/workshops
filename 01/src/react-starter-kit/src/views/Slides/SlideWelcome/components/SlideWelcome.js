//@flow
import React from 'react';
import styled from 'styled-components';

const Container = styled.div`
    font-family: Arial, Helvetica, sans-serif;
    display: flex;
    justify-content: center;
    flex-direction: column;
    align-items: center;
`;

const Title = styled.div`
    font-size: 7em;
`;

const SubTitle = styled.div`
    font-size: 3em;
    color: #808080;
    padding-top: 1em;
`;

const Separator = styled.div`
    margin: 4em;
    width: 75%;
    border: 0.035em solid #C8C8C8; //#264d73
`;

const Install = styled.div`
    font-size: 1.5em;
    color: #808080;
    padding-top: 1em;
`;

const SlideWelcome = () => (
    <Container>
        <Title>react-genova</Title>
        <SubTitle>workshop 01 - React Starter Kit</SubTitle>
        <Separator />
        <Install>{'git clone https://github.com/react-genova/workshops'}</Install>
        <Install>{'cd ./workshops/01/src/react-starter-kit'}</Install>
        <Install>{'npm install'}</Install>
        <Install>{'npm start'}</Install>
    </Container>
);

export default SlideWelcome;