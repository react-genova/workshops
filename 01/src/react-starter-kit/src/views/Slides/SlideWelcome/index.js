//@flow
export {default} from './components/SlideWelcome';