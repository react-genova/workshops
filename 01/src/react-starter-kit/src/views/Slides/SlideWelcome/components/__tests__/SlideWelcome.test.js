//@flow
import React from 'react';
import { shallow } from 'enzyme';
import SlideWelcome from '../SlideWelcome';

describe('Slide Welcome', () => {

    it('snapshot check', () => {
        const component = shallow(<SlideWelcome />);
        expect(component).toMatchSnapshot();
    });

});