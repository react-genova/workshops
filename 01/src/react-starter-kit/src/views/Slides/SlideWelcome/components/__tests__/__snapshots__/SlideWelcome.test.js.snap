// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Slide Welcome snapshot check 1`] = `
<styled.div>
  <styled.div>
    react-genova
  </styled.div>
  <styled.div>
    workshop 01 - React Starter Kit
  </styled.div>
  <styled.div />
  <styled.div>
    git clone https://github.com/react-genova/workshops
  </styled.div>
  <styled.div>
    cd ./workshops/01/src/react-starter-kit
  </styled.div>
  <styled.div>
    npm install
  </styled.div>
  <styled.div>
    npm start
  </styled.div>
</styled.div>
`;
